eventhandling/configureEvent.go

```diff
@@ -283,13 +260,43 @@ func (eh ConfigureMonitoringEventHandler) createPrometheusAlertsIfSLOsAndRemedia
 		alertingRulesConfig.Groups = append(alertingRulesConfig.Groups, alertingGroupConfig)
 	}
 
+	// create a new prometheus handler in order to query SLI expressions
+	prometheusHandler := utils.NewPrometheusHandler(
+		"",
+		&keptnv2.EventData{
+			Project: eventData.Project,
+			Service: eventData.Service,
+			Stage:   stage.Name,
+		},
+		nil,
+	)
+
+	// get SLI queries
+	projectCustomQueries, err := getCustomQueries(eh.keptnHandler, eventData.Project, stage.Name, eventData.Service)
+	if err != nil {
+		log.Println("Failed to get custom queries for project " + eventData.Project)
+		log.Println(err.Error())
+		return alertingRulesConfig, err
+	}
+
+	if projectCustomQueries != nil {
+		prometheusHandler.CustomQueries = projectCustomQueries
+	}
+
+	eh.logger.Info("Going over SLO.objectives")
+
 	for _, objective := range slos.Objectives {
+		eh.logger.Info("SLO:" + objective.DisplayName + ", " + objective.SLI)
+		// Get Prometheus Metric Expression
+		end := time.Now()
+		start := end.Add(-180 * time.Second)
```

Comment:
This is needed in order to mimic the behaviour of the old getSLIQuery function, which would always replace `$DURATION_SECONDS` with 180s 
```
query = strings.Replace(query, "$DURATION_SECONDS", "180s", -1)
```